app/models/user.rb

```diff
@@ -1,9 +1,13 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ]  Great job. Kindly remember, for the application to ask for confirmation of email Ensure `:confirmable` is enabled in this model as well. Kindly set that up. Don't forget to activate it as well in the devise migration file

Reply:
@tufoinnkuo10  Thanks for the review